src/www/ui/css/bootstrap-icons.css

```diff
@@ -0,0 +1,65 @@
+@font-face {
```

Comment:
Since bootstrap is MIT licensed, please add following comment to the top of the file to make it Reuse compliant.

```
SPDX-FileCopyrightText: © The Bootstrap Authors

SPDX-License-Identifier: MIT

File derived from https://github.com/twbs/icons/blob/main/font/bootstrap-icons.css
```